index.d.ts

```diff
@@ -29,6 +29,7 @@ declare namespace GSF {
         info(): Promise<ServiceInfo>;
         task(taskName: string): Task;
         tasks(): Promise<Array<Task>>;
+        taskInfoList(): Promise<Array<TaskInfo>>;
```

Comment:
What do you guys think it would take to refactor to a "starts with a verb" paradigm @gsf-smclamb & @gsf-sellis?